sys/core/src/TxsCore.hs

```diff
@@ -104,6 +104,7 @@ module TxsCore
 
   -- * LPE transformation
 , txsLPE
+, updateParams
```

Comment:
From the comments it seems that `updateParams` is related to the LPE transformation. Shouldn't this function be under the `Parameters` section above?

Reply:
Fixed